sections/semantics-forms.include

```diff
@@ -1080,52 +1310,50 @@
     </dd>
   </dl>
 
-  The <{label}> element <a>represents</a> a caption in a user interface. The
-  caption can be associated with a specific form control, known as the
-  <{label}> element's <dfn>labeled control</dfn>, either using the <{label/for}> attribute,
-  or by putting the form control inside the <{label}> element itself.
+  The <{label}> element <a>represents</a> a caption in a user interface. The caption can be
+  associated with a specific form control, known as the <{label}> element's
+  <dfn>labeled control</dfn>, either using the <{label/for}> attribute, or by putting the
+  form control inside the <{label}> element itself.
 
   Except where otherwise specified by the following rules, a <{label}> element has no
   <a>labeled control</a>.
 
   The <dfn element-attr for="label"><code>for</code></dfn> attribute may be specified to indicate a
   form control with which the caption is to be associated. If the attribute is specified, the
-  attribute's value must be the <a>ID</a> of a <a>labelable element</a> in the same {{Document}} as the
-  <{label}> element. If the attribute is specified and there is an
-  element in the {{Document}} whose <a>ID</a> is equal to the
-  value of the <{label/for}> attribute, and the first such element is a
-  <a>labelable element</a>, then that element is the <{label}>
-  element's <a>labeled control</a>.
+  attribute's value must be the <a>ID</a> of a <a>labelable element</a> in the same {{Document}}
+  as the <{label}> element. If the attribute is specified and there is an element in the
+  {{Document}} whose <a>ID</a> is equal to the value of the <{label/for}> attribute, and the
+  first such element is a <a>labelable element</a>, then that element is the <{label}> element's
+  <a>labeled control</a>.
 
-<div class="example">
+  <div class="example">
+    The following example showcases a <{label/for}> attribute, being used to associate a <{label}>
+    with a form control that is not a child element of the <{label}>.
+
+    <xmp highlight="html">
+      <form>
+        <div>
+          <label for="cname">
+            Customer name:
+          </label>
+          <input type="text" name="custname" id="cname">
+        </div>
+      </form>
+    </xmp>
 
-  The following example shows the use of a <{label/for}> attribute, to associate <{label}>s
-  which do not contain the element they label.
+    <p>
+      Note that the <{global/id}> attribute is required to associate the <{label/for}> attribute,
+      while the <{label/name}> attribute is required so the value of the input will be submitted
+      as part of the form.
+    </p>
+  </div>
 
-  <xmp highlight="html">
```

Comment:
putting labels in one column of a table and form controls in another is a classic use case for the `for` attribute. I think it would still be useful to highlight this case with a simple example.